css/main.css

```diff
@@ -116,6 +114,9 @@ kbd {
 .topbarSection.icon{
 	flex: 1 1 0%;
 }
+.topbarSection.leftmost{ /* The leftmost topbar choice doesn't have a black line so this adds it in. Having existing classes do this makes double thickness lines on other topbar options. */
+	border-left: 1px solid black;
```

Comment:
I don't like this, but i understand the need.
can we rename it to something easier to understand without the comment? like left-border?

Reply:
sure I can rename it. There's probably a way to do this better since we still have the issue of there being no left border on the second row of topbar elements when the screen width is smaller, but for now it was just a quick fix.